opentelemetry-instrumentation/README.rst

```diff
@@ -16,30 +16,69 @@ Installation
 
 This package provides a couple of commands that help automatically instruments a program:
 
+
 opentelemetry-instrument
 ------------------------
 
 ::
 
     opentelemetry-instrument python program.py
 
+The instrument command will try to automatically detect packages used by your python program
+and when possible, apply automatic tracing instrumentation on them. This means your program
+will get automatic distributed tracing for free without having to make any code changes
+at all. This will also configure a global tracer and tracing exporter without you having to
+make any code changes. By default, the instrument command will use the OTLP exporter but
+this can be overriden when needed.
+
+The command supports the following configuration options as CLI arguments and environment vars:
+
+
+* ``--exporter`` or ``OTEL_EXPORTER``
+
+Used to specify which trace exporter to use. Can be set to one or more
+of the well-known exporter names (see below) or a fully
+qualified Python import path to a span  exporter implementation.
```

Comment:
Does this mean I can set `exporter opentelemetry.exporter.jaeger:JaegerSpanExporter`?

Sorry for my misunderstanding, but when I look at the code it looks like it will error out if it doesn't match `ep.name` (where `ep` is an entry point).

```
exporters = {
    ep.name: ep for ep in iter_entry_points("opentelemetry_exporter")
}

for exporter_name in exporter_names:
    entry_point = exporters.get(exporter_name, None)
    if not entry_point:
        raise RuntimeError(
            "Requested exporter not found: {0}".format(exporter_name)
        )
```

So doesn't it _have_ to be `otlp`?

Reply:
You're right. This PR supported what the documentation says but I ended up removing it in order to simplify the implementation. Looks like I missed this piece of documentation when I removed the feature.